src/main/java/seedu/stockpal/data/product/Product.java

```diff
@@ -59,15 +60,21 @@ public boolean isPidMatch(Pid pid) {
     }
 
     public void increaseQuantity(Integer amountToChange) {
-        quantity.updateIncreaseQuantity(amountToChange);
+        try {
+            quantity.updateIncreaseQuantity(amountToChange);
+            Ui.printToScreen("Quantity updated. " + quantity.toString());
+        } catch (InventoryQuantityOverflowException IQOE) {
+            Ui.printToScreen("Overflow detected. Not change to quantity. " + quantity.toString());

```

Comment:
Typo: supposed to be "No change to quantity."

Reply:
Updated.